error-issue/src/error_monitor.ts

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright 2020 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {StackdriverApi} from './stackdriver_api';
+import {ErrorReport, Stackdriver} from './types';
+
+import fetch from 'node-fetch';
+import statusCodes from 'http-status-codes';
+
+const ERROR_ISSUE_REPORT_ENDPOINT =
+  'https://us-central1-amp-error-issue-bot.cloudfunctions.net/error-issue';
```

Comment:
Oh, it's a const here. Hmm.... merge please 🤔

Reply:
Done.